src/shared/api/urls.ts

```diff
@@ -206,6 +206,18 @@ export function getOncoKbApiUrl() {
     return buildCBioPortalAPIUrl(`proxy/oncokb`);
 }
 
+export function getInstituteLogoUrl() {
+    if (AppConfig.serverConfig.skin_right_logo) {
+        if (/^http/.test(AppConfig.serverConfig.skin_right_logo)) {
+            return AppConfig.serverConfig.skin_right_logo;
+        } else {
+            return `/images/${AppConfig.serverConfig.skin_right_logo}`;
```

Comment:
@alisman this should prolly be relative to root url of app for portals running in subfolder, e.g. pdx.cbioportal.org/pdx

Reply:
ah, yeah, didn't think about that